### ISSUE TITLE
SQL Server Support Full Text Search

### ISSUE THREAD

**Issue by dleggen** [dotnet/aspire#6501]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

It would be nice to be able to use the full text search feature that is available on SQL server. 

### Describe the solution you'd like

Be able to enable full text search so that the indexes can be added via a migration.

`CREATE FULLTEXT CATALOG ftCatalog AS DEFAULT;
CREATE FULLTEXT INDEX ON Posts(Content) KEY INDEX PK_Posts; `

**Comment by pfo-omicsstudio** [comment]
Agreed, just ran into this today, no full text search in SQL server is a massive problem considering it IS supported in Azure.

**Comment by davidatone** [comment]
Has there been a decent workaround for this? Currently, I am building my own Docker image that has it enabled and using that.

**Comment by Julien-Marpault** [comment]
Any news about it ?

**Comment by Cripacx** [comment]
I'm also looking forward to it.

**Comment by rbev** [comment]
It would be fantastic for this to be supported, it defeats the purpose if I have to manage my own sql server image.

Additionally;
#6550 was closed as a dupe of itself instead of linking here.

**Comment by warwickschroeder** [comment]
Could there please be some movement on this? It would be extremely useful.
